### PR TITLE
Mm 37504 - add new invite people button to sidebar channel dropdown

### PR DESCRIPTION
**Pull request by pablovelezvidal**
#### Summary
This PR adds a new invite people button to sidebar channel dropdown

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-37504

#### Screenshots
Captura de pantalla 2021-08-11 a las 16 00 22

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: components/sidebar/add_channel_dropdown.tsx [hunk 1]
```diff
@@ -74,14 +88,15 @@ class AddChannelDropdown extends React.PureComponent<Props, State> {
             <Menu.ItemAction
                 id={'browseDirectMessages'}
                 onClick={this.props.handleOpenDirectMessagesModal}
-                icon={<i className='icon-account-plus-outline'/>}
+                icon={<i className='icon-account-outline'/>}
```
Comment (marianunez) [inline components/sidebar/add_channel_dropdown.tsx:91]: Is this an intended change?

**Review by marianunez — APPROVED** [review]
LGTM

File: components/sidebar/add_channel_dropdown.tsx (hunk 1 above)
Comment (pablovelezvidal) [reply components/sidebar/add_channel_dropdown.tsx:91]: yes! it changed to give the invite people button  the plus-outline.

Comment (marianunez) [reply components/sidebar/add_channel_dropdown.tsx:91]: Oh I see, we gave the new menu item the plus and removed it from the DM. Gotcha!

**Review by matthewbirtch — CHANGES_REQUESTED** [review]
@pablovelezvidal Looks like there is a layout problem on small screens with the mobile webview for this menu
image

Other than that, I think it's good to go

**Comment by pablovelezvidal** [comment]
Fixed @matthewbirtch 

Captura de pantalla 2021-08-11 a las 21 15 26

**Review by neallred — APPROVED** [review]
looks good to me!

**Review by matthewbirtch — APPROVED** [review]
thanks @pablovelezvidal

**Pull request merged**